### PR TITLE
Changed StrictVersion checking of setuptools to LooseVersion

### PR DESCRIPTION
**Pull request by raddessi**
### What does this PR do?
Fixes an installation issue in conda envs

### What issues does this PR fix or reference?
https://github.com/saltstack/salt/issues/56184

### Previous Behavior
Install doesnt work

### New Behavior
Install works

### Tests written?
No

### Commits signed with GPG?
No... I know I should know better it's on my list

### CODE REVIEW

**Review by Ch3LL — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -701,7 +701,7 @@ def finalize_options(self):
 
     def run(self):
         from distutils.version import StrictVersion
```
Comment (Ch3LL) [inline setup.py]: actually would you mind removing this import, looks like it wont be used anymore.

**Review by Ch3LL — CHANGES_REQUESTED** [review]

**Review by s0undt3ch — CHANGES_REQUESTED** [review]
As Megan pointed out. Let's remove the unused import.

**Comment by raddessi** [comment]
Oh nice, i didnt catch that. Will do

Hey should i open a separate issue asking about if you all are open to setting up black/precommit to auto format and check for stuff like this automatically?

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by s0undt3ch** [comment]
We will `black`en salt as soon as we start merging into master.
Lint should have caught this but we might have disabled the check somewhere.

**Comment by raddessi** [comment]
Will 3000.1 be the first merge to master?

**Review by s0undt3ch — APPROVED** [review]

**Review by Ch3LL — APPROVED** [review]

**Comment by Ch3LL** [comment]
> Will 3000.1 be the first merge to master?

I'm not sure what you are asking? 3000.1 release will be tagged from the master branch if that is what you are asking.

**Comment by raddessi** [comment]
Ah sorry I thought you all were going to do a mass merge of all the items marked for 3000.1 and then tag it, I must misunderstand something, I'll re read the new merge strategy doc. Thanks again

**Comment by Ch3LL** [comment]
Oh no worries, i think i just misunderstood the question. This is the project board we are using to track what issues need to get in for 3000.1, looks like your issue is under commit, so looks like we will get this one in :)

**Comment by Ch3LL** [comment]
errr sometimes it helps to include links you refer to lol https://github.com/saltstack/salt/projects/8

**Comment by raddessi** [comment]
Totally, I got what you meant. Yes that was the board I was thinking of also :) thanks

**Review by dwoz — APPROVED** [review]

**Pull request merged**